test/jdk/javax/management/remote/mandatory/subjectDelegation/SubjectDelegation2Test.java

```diff
@@ -72,7 +72,7 @@ public static void main(String[] args) throws Exception {
             //
             System.out.println("Start RMI registry...");
             Registry reg = null;
-            int port = 5800;
+            int port = 5880;
             while (port++ < 6000) {
```

Comment:
Does the loop need to terminate on 5900 instead of 6000?

Reply:
Quite right.